IoTConnect/IoTConnect/Utilities/MQTT copy/CocoaMQTT.h

```diff
@@ -0,0 +1,17 @@
+//
```

Comment:
Why two folder are added "MQTT copy" and "MQTT"?

Reply:
Deleted MQTT copy